docs/en/ingest-management/commands.asciidoc

```diff
@@ -761,6 +789,15 @@ Install {agent} in a non-interactive mode. This flag is helpful when
 using automation software or scripted deployments. If {agent} is
 already installed on the host, the installation will terminate.
 
+`--privileged`::
+Run {agent} with full superuser privileges.
+This is the usual, default running mode for {agent}.
+The `--privileged` option allows you to switch back to running an agent with full administrative privileges when you have been running it in `unprivileged`.
+
+Changing from `unprivileged` to `privileged` mode is supported only on Linux systems. 
```

Comment:
Switching between privileged and unprivileged works on all OS's.

Reply:
I've removed the restriction about the switch being supported only on Linux.